pynetbox/core/query.py

```diff
@@ -211,6 +212,16 @@ def __init__(
             In (e.g. /api/dcim/devices/?name='test') 'name': 'test'
             would be in the filters dict.
         * **key** (int, optional): Database id of the item being queried.
+        * **expect_json** (bool, optional): If True, expects JSON response
+            and sets appropriate Accept header. If False, expects raw content
+            (e.g., SVG, XML) and returns text. Defaults to True.
+
+        ## Note
```

Comment:
Is this supposed to be here, doesn't seem part of this PR?

Reply:
This is my note when exploring this part of the codebase. I am not entirely positive if I understand this correctly tho.
I tried to add an attribute and realize the `count` attribute was not declared but used, but it actually suppose to be that way.